d2bs/kolbot/libs/bots/Diablo.js

```diff
@@ -443,6 +444,9 @@ function Diablo() {
 		if (Config.PublicMode) {
 			Pather.makePortal();
 			say(Config.Diablo.EntranceTP);
+			if (Config.Diablo.StartWalking) {
```

Comment:
Don't need if (...)
`Pather.teleport = !Config.Diablo.WalkClear;`

Reply:
If Pather.teleport has been disabled by the profile and they also have Config.Diablo.WalkClear = false, this would accidentally turn Pather.teleport = true.

Reply:
Good point, change to
`Pather.teleport = !Config.Diablo.WalkClear && Pather._teleport;`

if WalkClear is false, _teleport is true -> true
if WalkClear is true, _teleport is true -> false
if WalkClear is false, _teleport is false -> false
if WalkClear is true, _teleport is false -> false